examples/otlp/README.md

```diff
@@ -10,6 +10,11 @@ SDK](https://github.com/open-telemetry/opentelemetry-cpp). The application then
 calls a `foo_library` which has been instrumented using the [OpenTelemetry
 API](https://github.com/open-telemetry/opentelemetry-cpp/tree/main/api).
 
+To enable TLS authentication for OTLP grpc exporter, SslCredentials can be used by
+specifying the path to client certificate pem file via OtlpExporterOptions. The path to such a
```

Comment:
```suggestion
specifying the path to client certificate pem file, or the string containing this certificate via OtlpExporterOptions. The path to such a
```

Reply:
Sorry for one last comment :)

Reply:
no worries, makes sense, forgot to update.